src/config/dynamic-theme-fixes.config

```diff
@@ -585,6 +585,27 @@ body[style] {
 
 ================================
 
+bestbuy.ca
+
+INVERT
+.centerContainer
+
+CSS
+[class^="facetName"], [class^="title"], [class^="sortLabel"], [class^="headerText"], [class^="subTitle"] {
+    filter:invert(100%) hue-rotate(180deg) contrast(90%) !important;
+}
+.backgroundContainer_ryHAa {
+    z-index:0 !important;
+}
+.featureBanner_8BLDb .centerContainer {
```

Comment:
Where can I find this element?

Reply:
https://www.bestbuy.ca/en-ca/category/gaming-monitors/474835
In order to make the banner appear (in this case, the two monitors), the z-index of this element must be increased, along with the **.featureBanner_8BLDb .centerContainer** element.

Reply:
In my testing, if I did not specify this exact selector, it did not work.

`when bestbuy pushes a CSS update it might change` Yes, that is true. As it is for all custom CSS in this config file.

Reply:
>  Yes, that is true. As it is for all custom CSS in this config file.

I meant, that this looks like discord.com First a normal string and than randomized number and every update it changes the numbers.

> In my testing, if I did not specify this exact selector, it did not work.

```CSS
div[class^="featureBanner"] .centerContainer
```